policyengine_us/parameters/calibration/ssdi/outlay_projections.yaml

```diff
@@ -0,0 +1,18 @@
+description: Projections of total federal outlayers for SSDI (2022-2032)
+metadata: 
+  unit: currency-USD
+  reference:
+  - title: Baseline Projections - Social Security Disability Insurance | Congressional Budget Office
+    href: https://www.cbo.gov/system/files?file=2022-05/51307-2022-05-ssdi.pdf
+values:
+  2022-01-01: 141_400_000_000
```

Comment:
Could we re-add the historical values here?

Reply:
I didn't find the historical ssdi data on the website of Congressional Budget Office.